live-examples/css-examples/rhythm/line-height-step.css

```diff
@@ -0,0 +1,20 @@
+.example-container {
+    background-image: repeating-linear-gradient(180deg, #ccc, #ccc 20px, #dbdbdb 20px, #dbdbdb 40px);
+    border: .75em solid;
+    padding: .75em;
+    text-align: left;
+    width: 80%;
+    line-height: normal;
+    font-size: 20px;
+    display: grid;
+    grid-gap: 20px;
+    grid-template-columns: 1fr 2fr;
+}
+
+.example-container h2, p {
+    margin: 0;
+}
+  
+.example-container h2 {
+    font-size: 33px;
```

Comment:
This overflows vertically quite easily - at about 1000 pixels wide it looks like:

<img width="1010" alt="screen shot 2018-03-05 at 7 53 59 pm" src="https://user-images.githubusercontent.com/432915/37013496-0e24857a-20af-11e8-9740-351054fbcc87.png">

Could we reduce the font size and/or have a shorter text?